samples/metaapp/build.gradle

```diff
@@ -125,20 +128,6 @@ dependencies {
     //gson
     implementation 'com.google.code.gson:gson:2.8.5'
 
+    // TODO remove this dependency
```

Comment:
@carmelo-iriti when can we remove this dependency?

Reply:
@andresilveirah to remove this dependency we need to refactor the component that is using it